cli/src/main.rs

```diff
@@ -121,21 +129,26 @@ async fn run_test(test_args: TestArgs) -> anyhow::Result<i32> {
         repo_head_commit_epoch.clone(),
     )?;
 
-    if junit_paths.is_empty() {
-        return Err(anyhow::anyhow!("No junit paths provided."));
+    if junit_paths.is_empty() && bazel_bep_path.is_none() {
+        return Err(anyhow::anyhow!("No junit or bazel BEP paths provided."));
```

Comment:
I think I'd rather us keep the old error message until we document this feature (even if it'll appear in `--help`)